### ISSUE TITLE
Tutorial: Crowdsourced Sentiment Analysis

### ISSUE THREAD

**Issue by larskarg** [HazyResearch/snorkel#970]
Hi, great package and very interesting idea. I played around with [Crowdsourced_Sentiment_Analysis.ipynb](https://github.com/HazyResearch/snorkel/blob/master/tutorials/crowdsourcing/Crowdsourced_Sentiment_Analysis.ipynb) a bit using the latest version of Snorkel and Python3. Besides some minor changes necessary to support Python3, I saw that the `score` function was refactored (session object was removed) in the context of issue https://github.com/HazyResearch/snorkel/issues/569 and moved to `Classifier->score`? I made the necessary [changes to the notebook](https://github.com/larskarg/snorkel/blob/master/tutorials/crowdsourcing/Crowdsourced_Sentiment_Analysis.ipynb). After that, and when I ran the updated notebook, I saw a drop in `Accuracy` (last cell). To me it looks like that could be related to `TextRNN` since I saw some `UserWarnings` (2nd last cell) running `TextRNN`: 

```
learning/disc_models/rnn/rnn_base.py:40: UserWarning: Candidate 557 has 
argument past max length for model:	[arg ends at index 28; max len 28]
  warnings.warn('\t'.join([w.format(i), info]))
```

I was wondering what I could have possibly done wrong here? Do I have to make further changes to the notebook to run it with the latest version of Snorkel and Python3? Any thoughts? Any insight would be appreciated.

Thanks,
Lars

**Comment by ajratner** [comment]
Hey @larskarg thanks for looking into this!  Do you think you could submit a PR so we can check out the diff and see what's going on here?  thanks!

**Comment by larskarg** [comment]
hi @ajratner thanks for looking into it. Highly appreciated. Sure, [PR](https://github.com/HazyResearch/snorkel/pull/973) created. I just realized that a new version (`0.7-beta`) was released. I tested my changes with the previous release (`0.6.3`). If I have time tomorrow I will give `0.7-beta` a try.

**Comment by ajratner** [comment]
Thanks!! And yeah sorry I forgot to mention- definitely let us know if 0.7
takes care of it

**Comment by larskarg** [comment]
Unfortunately, I only see a slight accuracy improved (from `0.5625` to `0.625`) but the value is still lower than the one in the [original notebook](https://github.com/HazyResearch/snorkel/blob/master/tutorials/crowdsourcing/Crowdsourced_Sentiment_Analysis.ipynb) (`0.703125`).  [PR](https://github.com/HazyResearch/snorkel/pull/973) updated to support `v0.7-beta`. Warning also persists:
```
snorkel/lib/python3.6/site-packages/snorkel/learning/tensorflow/rnn/rnn_base.py:36:
UserWarning: Candidate 557 has argument past max length for model:	[arg ends 
at index 28; max len 28]  warnings.warn('\t'.join([w.format(i), info]))
```

**Comment by tienhotze** [comment]
@larskarg I got exactly same accuracy results for all the cells except for the last one for lstm.score(test_cands, test_cand_labels) where i get these results instead (running snorkel 0.7):
Accuracy: 0.5625
Number correct: 36
Number incorrect: 28

I also got the same warnings as you did.

**Comment by ajratner** [comment]
@paroma assigned to you since you're looking at the PRs!

**Comment by paroma** [comment]
fixed with #973
